chemie/customprofile/templates/customprofile/add_card.html

```diff
@@ -42,7 +42,7 @@ <h1 class="title is-2">Legg til studentkort</h1>
                 </div>
                 <div class="column is-4 is-hidden-mobile"></div>
             </div>
-          <input type="submit" class="button" name="addStudentcard" value="Legg til studentkortnummer" height="10px"/>
+            <input type="submit" class="button" name="addStudentcard" value="Legg til studentkortnummer" height="10px"/>
```

Comment:
Skulle gjerne likt å ha en modal med "bekret endringer av kort" her hvis brukeren alt har et kortnummer liggende inn, for nå endreres kortnummeret uten å si ifra til brukeren.

Reply:
Skjønner. 
anken bak hvordan det er nå, er at å legge til kort skal skje fort. Det vil nok ikke sinke prosessen så mye å legge til en modal, men da kommer vi til det andre poenget, som er: Dersom man blir redirectet til add_card (profile/add_card) så er ikke kortet man har scannet lagret i databasen i utgangspunktet. At kortnr blir endret er derfor ikke noe "dumt" da kortnr som ligger inne uansett vil være feil.